korge/src/commonMain/kotlin/korlibs/korge/view/CachedContainer.kt

```diff
@@ -84,16 +84,21 @@ open class CachedContainer(
         val cache = _cacheTex!!
         ctx.refGcCloseable(cache)
 
-        val renderScale: Float = when (ctx.views?.gameWindow?.quality) {
+        val renderScale: Float = when (ctx.quality) {
             GameWindow.Quality.PERFORMANCE -> 1f
             else -> ctx.devicePixelRatio
         }
         //val renderScale = 1.0
 
+        val doExpensiveScaling = expensiveScaling ?: when(ctx.quality) {
+            GameWindow.Quality.PERFORMANCE -> false
+            else -> true
+        }
```

Comment:
Do you think that by adding some properties the Quality enum could we avid somehow the when so the code is simpler?